tests/test_filesize.py

```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
```

Comment:
There are different file size requirements for different file types. I was expecting to create multiple tests for different groupings of file size expectations. I'm not sure how adding the file size to the prefs.ini would help- but we can discuss face to face.